src/libraries/Common/src/Roslyn/ISyntaxHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.DotnetRuntime.Extensions
+{
+    internal interface ISyntaxHelper
```

Comment:
this is technically not necessary as it serves to abstract over C#/VB (and runtime doesn't need that).  However, i wanted to keep our code closely in sync with theirs to make porting any changes over easy.

Reply:
The whole file isn't necessary?

Reply:
it's necessary in that it's what the code i wrote compiles against.  it serves, however, as an abstrcation over VB and C# so that your code *could* work on VB if you wanted it to.  Since you don't need VB support, thsi could technically be greatly simplified.  but at the cost of major divergence from the code in roslyn (which could make things like portings fixes/changes harder).